hack/make-rules/local-up-openyurt.sh

```diff
@@ -64,7 +64,7 @@ readonly LOCAL_ARCH=$(go env GOHOSTARCH)
 readonly LOCAL_OS=$(go env GOHOSTOS)
 readonly CLUSTER_NAME="openyurt-e2e-test"
 readonly KUBERNETESVERSION=${KUBERNETESVERSION:-"v1.22"}
-readonly NODES_NUM=${NODES_NUM:-2}
+readonly NODES_NUM=${NODES_NUM:-3}
```

Comment:
Do not set default value of `NODES_NUM`, because we originally use 2 nodes to run current e2e tests. If you want to run autonomy test with 3 nodes, you can explicitly set `NODES_NUM=3` and then run `local-up-openyurt.sh`.